src/examples/element_matrix_redecomp.cpp

```diff
@@ -4,7 +4,7 @@
 // SPDX-License-Identifier: (MIT)
 
 /**
- * @file matrix_redecomp.cpp
+ * @file element_matrix_redecomp.cpp
  *
  * @brief Demonstrates moving a matrix using redecomp::MatrixTransfer
```

Comment:
Consider "Demonstrates moving element matrices using redecomp::MatrixTransfer"; this may be a little more precise since now we have multiple definitions of what "a matrix" is.